src/admin/components/SearchForm.js

```diff
@@ -1,31 +1,31 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 
+import { SearchFormBase } from 'amo/components/Search/SearchForm';
```

Comment:
I'd like to avoid the different apps importing from each other if possible. Should the base go in core instead? That may alleviate the need to namespace under `Search/` as well.